src/io/flutter/logging/FlutterLogEntryParser.java

```diff
@@ -32,6 +43,35 @@ public static FlutterLogEntry parse(@Nullable String id, @Nullable Event event)
       switch (id) {
         case LOGGING_STREAM_ID:
           return parseLoggingEvent(event);
+        case VmService.GC_STREAM_ID:
+          final IsolateRef isolateRef = event.getIsolate();
+          final HeapMonitor.HeapSpace newHeapSpace = new HeapMonitor.HeapSpace(event.getJson().getAsJsonObject("new"));
+          final HeapMonitor.HeapSpace oldHeapSpace = new HeapMonitor.HeapSpace(event.getJson().getAsJsonObject("old"));
+
+          // TODO(devoncarew): update the VM library - timestamp is a long
+          final long timestamp;
+
+          if (event.getJson().get("timestamp") == null) {
```

Comment:
Do you mean:

    if (event.getJson().get("timestamp") != null)

Also, different (though maybe no better):

    if (event.getJson().has("timestamp"))

Reply:
yes, either of these are better then the current check :)

Reply:
updated